pkg/template/templateprocessing/template_test.go

```diff
@@ -249,7 +249,7 @@ func TestEvaluateLabels(t *testing.T) {
 				]
 			}`,
 			Output: `{
-				"kind":"Template","apiVersion":"template.openshift.io/v1","metadata":{"creationTimestamp":null},
```

Comment:
Yes, this is expected because the omitzero field tag was added to metav1.ObjectMeta's CreationTimestamp.